src/devtools/store.js

```diff
@@ -351,6 +353,11 @@ export default class Store extends EventEmitter {
           break;
         }
         const child = ((this._idToElement.get(childID): any): Element);
+
+        // We intentionally ignore collapsed state when determining an item's index.
+        // We only do this for the direct path to an element.
+        // That's because the index of the element is meaningless if it's inside of a collapsed tree.
+        // If this index is used to display the element, the caller should also un-collapse its ancestors.
         index += child.weight;
```

Comment:
Wait, doesn’t this need to honor collapsed because it counts previous siblings of ancestors, not just ancestors?

The root weight calculation later in the function too.

Reply:
> Wait, doesn’t this need to honor collapsed because it counts previous siblings of ancestors, not just ancestors?

Hmm...yeah, that was my initial intuition as well, but then I convinced myself it wasn't true. I think you're right though that something might be off here. Lemme look at it a bit.

> The root weight calculation later in the function too.

Hmm... roots themselves are invisible. They will never be collapsed.

Reply:
Yeah I think I may have done the _exact opposite_ thing here. 😄 I really need to write some good tests for this (#115)

Reply:
Okay. I reverted that misguided bit. Nice catch 😄